systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -2064,6 +2073,12 @@ void testLabelsAndAnnotationForPVC() {
             .endSpec()
             .done();
 
+        LOGGER.info("Check if Kubernetes labels are applied");
+        Map<String,String> actualStatefulSetLabels = KafkaResource.kafkaClient().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get()
+                .getSpec().getKafka().getTemplate().getStatefulset().getMetadata().getLabels();
+        assertThat(actualStatefulSetLabels, is(statefulSetLabels));
```

Comment:
Does this work? I thought that the `app.kubernetes.io/managed-by` should not be configurable by user.

Reply:
TBH I found that on Kubernetes website and tried that -> both labels were set

Reply:
@scholzj is it some kind of issue or is it correct?